doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -20,7 +18,7 @@ Procedure
 **********
 The procedure to install CORTX on OVA is mentioned below.
 
-#. Download and unzip the `cortx-va-1.0.2.zip <https://github.com/Seagate/cortx/releases/download/VA/cortx-va-1.0.2.zip>`_ file from `our release page <https://github.com/Seagate/cortx/releases/tag/VA>`_. This contains the virtual machine image.
+#. Download and unzip the `cortx-va-1.0.3.ova <https://github.com/Seagate/cortx/releases/download/ova-1.0.3/cortx-va-1.0.3.ova>`_ file from `our release page <https://github.com/Seagate/cortx/releases/tag/ova-1.0.3>`_. This contains the virtual machine image.
```

Comment:
Can we use https://github.com/Seagate/cortx/releases/latest . So that link will always point to the latest release and we do not need to change the link for any new release.

Also previously @johnbent mentioned to keep all our Releases as `pre-release` and add the text mentioned in https://github.com/Seagate/cortx/releases/tag/VA

Reply:
done